### PR TITLE
Improv

### PR DESCRIPTION
**Pull request by Nagasaki45**
I added some improvements:
- My Kobo Glo have no default monospace font. I'm used to embed UbuntuOne into the epub to solve this issue.
- The cross links was broken, and failed my eReader from loading the book at all.
- The rest are minor css improvements.

I would like to hear what you think.

### CODE REVIEW

**Comment by schmittlauch** [comment]
@Nagasaki45 Thanks for the effort with the cross links, I also encountered this but hadn't yet found the time to fix it. As you seem to have a grasp about css: Can you tell me why
```css
pre {
  margin: 10pt;
}
```
makes the code examples wrap? I included `white-space: pre-wrap;` to get the code samples to wrap and I needed to experiment quite long, but your version also works.

**Comment by Nagasaki45** [comment]
Hi @schmittlauch. I'm not a CSS expert at all. I too modified the CSS by trial and error until I've got something working. Notice that the `white-space: pre-wrap;` just moved from the `pre` block to the `code` block, so it is still applied to the code snippets.
It might be a good idea to validate my CSS changes with someone that is more familiar with the subject...

File: epub-stylesheet.css [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
-h4 {
-  text-align: left;
-}
-h5 {
-  text-align: left;
-}
-h6 {
-  text-align: left;
+  font-family: "Ubuntu Mono", monospace;
+  font-size: 0.88em;
```
Comment (pvorb) [inline epub-stylesheet.css]: Why the smaller font size?

File: en/13-for-a-few-monads-more.md [hunk 2]
```diff
@@ -738,8 +736,8 @@ Reader? Ugh, not this joke again.
 
 ![bang youre dead](img/revolver.png)
 
-In the [chapter about
-applicatives](functors-applicative-functors-and-monoids), we saw that
+In the [chapte about
```
Comment (pvorb) [inline en/13-for-a-few-monads-more.md]: You introduced an error here.

**Pull request merged**